CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 4.0.0-dev.1
+- This package now targets package:lints for the generated code. The generated
+code uses C symbol names as is. Use either `// ignore_for_file: lintRule1, lintRule2`
+in the `preamble`, or rename the symbols to make package:lints happy.
+- Function pointers are now named as `_<func>Ptr` instead of `_<func>_ptr`.
```

Comment:
This is not visible in the public API right? I think we can omit it.

Reply:
Done.